farmhelden/rest_server/models.py

```diff
@@ -18,10 +18,18 @@ class User(models.Model):
     user_type = models.CharField(max_length=1, choices=USER_CHOICES, null=True)
 
 
+class Location(models.Model):
+    LOCATION_TYPES = (('1', 'Bauernhof'), ('2', 'Feld'))
+    id = models.AutoField(primary_key=True)
+    point = PointField()
+    info = models.TextField(max_length=1000)
+    location_type = models.CharField(max_length=1, choices=LOCATION_TYPES)
+
+
 class Farm(models.Model):
     id = models.AutoField(primary_key=True)
     user_id = models.ForeignKey(User, on_delete=models.CASCADE)
-    point = PointField()
+    location = models.ForeignKey(Location, on_delete=models.CASCADE)
```

Comment:
Hat eine Farm möglicherweise mehrere Locations? Verschiedene Felder?

Reply:
Hatte die Location in der Farm eigentlich so vorgesehen dass sie nur den Standort angibt. Aber wir können die Farm auch als Sammelbegriff für alle Felder die zu einem Bauern gehören verwenden.

Reply:
Klingt für mich sinnvoll!